tests/bsim/bluetooth/host/l2cap/multilink_peripheral/src/dut.c

```diff
@@ -251,7 +251,7 @@ static void start_advertising(void)
 
 	LOG_DBG("starting advertiser");
 
-	err = bt_le_adv_start(BT_LE_ADV_CONN_ONE_TIME, ad, ARRAY_SIZE(ad), NULL, 0);
+	err = bt_le_adv_start(BT_LE_ADV_CONN_FAST_2, ad, ARRAY_SIZE(ad), NULL, 0);
```

Comment:
In other cases you've replace `BT_LE_ADV_CONN_ONE_TIME` with `BT_LE_ADV_CONN_FAST_1` - Why `BT_LE_ADV_CONN_FAST_2` here?

Reply:
I don't have any preference for one over the other. I'll switch them all to `BT_LE_ADV_CONN_FAST_1` for consistency.

Reply:
`BT_LE_ADV_CONN_ONE_TIME` uses the timings of `BT_LE_ADV_CONN_FAST_2`. So replacing `BT_LE_ADV_CONN_ONE_TIME` with `BT_LE_ADV_CONN_FAST_2` would be the refactor with no change. OTOH, `BT_LE_ADV_CONN_FAST_1` will potentially connect faster. I doubt the choice in the tests was made consciously, so I think we can use the faster one.